subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/AnnotationProcessingCompileTask.java

```diff
@@ -132,14 +133,6 @@ private URLClassLoader createProcessorClassLoader() {
         }
     }
 
-    private Processor instantiateProcessor(Class<?> processorClass) {
-        try {
-            return (Processor) processorClass.getConstructor().newInstance();
-        } catch (Exception e) {
-            throw new IllegalArgumentException("Could not instantiate annotation processor '" + processorClass.getName() + "'");
```

Comment:
We are loosing a bit of error context here, i.e. that we wanted to instantiate the annotation processor. I guess the class is still visible in the wrapped exception.

Reply:
Yes... I made the mistake when I replaced all `getConstructor().newInstance()` back to `ReflectionUtil.newInstance`... this code is originally `getConstructor().newInstance()` and shouldn't be changed at all.. thanks a lot :-)